lib/tessel/deploy.js

```diff
@@ -10,6 +10,7 @@ var fsTemp = require('fs-temp');
 var browserify = require('browserify');
 var uglify = require('uglify-js');
 var glob = require('glob');
+var rimraf = require('rimraf');
```

Comment:
TODO: `rimraf` can be removed across the project, in favor of `fs-extra`, which has more capabilities—including an `rm -rf` operation. @rwaldron